src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const costs = 40;
+  let amount = 0;
+  const discount1 = 20;
+  const discount2 = 50;
+
+  if (days < 3) {
+    amount = days * costs;
+  } else if (days < 7) {
+    amount = days * costs - discount1;
+  } else {
+    amount = days * costs - discount2;
+  }
+
+  return amount;
```

Comment:
1) it'd better to use early return
2) instead of discount1 and discount2 we can use more meaningful names
```suggestion
  const rentalCost = days * 40
  const baseDiscount = 20;
  const proDiscount = 50;
  
  if (days >= 7) {
    return rentalCost - proDiscount;
  }
  
  if (days >= 3) {
    return rentalCost - baseDiscount;
  } 
  
  return rentalCost 
  ```